### ISSUE TITLE
Tooltips?

### ISSUE THREAD

**Issue by joerg-rade** [rjaros/kvision#28]
Is there an example for tooltips (aka: popup / bubble help)?

**Comment by rjaros** [comment]
Tooltips are the thing I obviously missed when implementing Bootstrap components :-( 
It is possible to use them at the moment but it requires some hacks. I will try to add standard support for tooltips in the next version. For now just use `title` attribute (the same attribute will be used as the tooltip source).

**Comment by rjaros** [comment]
Support for tooltips (and popovers) is included in KVision 0.0.32. Sample usage:
```
button("Button") {
    title = "A tooltip"
    enableTooltip()
}
```
